modules/sandboxed-containers-create-kataconfig-resource-web-console.adoc

```diff
@@ -0,0 +1,74 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/deploying_sandboxed_containers.adoc
+
+[id="sandboxed-containers-create-kataconfig-resource-web-console_{context}"]
+= Creating the KataConfig custom resource in the web console
+
+You must create one `KataConfig` custom resource (CR) to enable installing the `kata` runtime on your cluster nodes.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the {sandboxed-containers-operator}.
+
+[NOTE]
+====
+The Kata runtime is installed on all worker nodes by default. To install the `kata` runtime only on selected nodes, you must label the nodes first, and then create the `KataConfig` CR and include the label in the YAML file.
+====
+
+.Procedure
+
+. From the *Administrator* perspective in the web console, navigate to *Operators* → *Installed Operators*.
+
+. Select the {sandboxed-containers-operator} from the list of operators.
+
+. In the *KataConfig* tab, click *Create KataConfig*.
+
+. In the *Create KataConfig* page, select to configure the `KataConfig` CR via *YAML view*.
+
+. Copy and paste the following manifest into the YAML view:
+
++
+[source,yaml]
+----
+apiVersion: kataconfiguration.openshift.io/v1
+kind: KataConfig
+metadata:
+  name: cluster-kataconfig
+----
++
+If you want to install the `kata` runtime only on selected nodes, include the label in the manifest:
+
++
+[source,yaml]
+----
+ apiVersion: kataconfiguration.openshift.io/v1
+ kind: KataConfig
+ metadata:
+   name: cluster-kataconfig
+ spec:
+   kataConfigPoolSelector:
+     matchLabels:
+     <label_key>: '<label_value>'
+----
+
+. Click *Create*.
+
+The new `KataConfig` CR is created and begins to install the `kata` runtime on the worker nodes.
+
+[IMPORTANT]
+====
+{sandboxed-containers-first} installs `kata` only as a secondary optional runtime on the cluster and not as the primary runtime.
+====
+
+.Verification
+
+. In the *KataConfig* tab, select the new KataConfig CR.
+
+. In the *KataConfig* page, select the *YAML* tab.
+
+. Monitor the *installationStatus* field in the status.
```

Comment:
Users can also add `oc get kubeconfig -o wide` and look at the status field. 

cc @jensfr @bpradipt

Reply:
@zanetworker -since this procedure is dealing with the web console procedure, I'm not sure we want to add a CLI procedure...

Reply:
Sure, can we add it as a CLI procedure?

Reply:
@zanetworker in the CLI procedure we have `watch oc describe kataconfig` for the verification, where the user can follow the installation process, and once the value of Completed nodes equals the number of worker or labeled nodes, they know the installation is complete. 

Is that alright? Should it be different?

Reply:
Let me quickly answer for @zanetworker : on the UI, there is no "auto-refresh" button. What you get is a message that teh object was updated, and a "Reload" button to get the updated version.

Reply:
I added this as a step in the Verification process. Thanks!